addons/web/static/tests/core/debug/debug_manager_tests.js

```diff
@@ -720,4 +720,58 @@ QUnit.module("DebugMenu", (hooks) => {
             ]
         );
     });
+
+    QUnit.test("set defaults: set scheduled date", async (assert) => {
+        assert.expect(3);
+
+        prepareRegistriesWithCleanup();
+        patchWithCleanup(odoo, {
+            debug: true,
+        });
+
+        registry.category("services").add("user", makeFakeUserService());
+        registry.category("debug").category("form").add("setDefaults", setDefaults);
+
+        const serverData = getActionManagerServerData();
+        serverData.actions[1234] = {
+            id: 1234,
+            xml_id: "action_1234",
+            name: "Partners",
+            res_model: "partner",
+            res_id: 1,
+            type: "ir.actions.act_window",
+            views: [[18, "form"]],
+        };
+        serverData.views["partner,18,form"] = `
+            <form>
+                <field name="__last_update"/>
+            </form>`;
+        serverData.models["ir.ui.view"] = {
+            fields: {},
+            records: [{ id: 18 }],
+        };
+        serverData.models.partner.records = [{ id: 1, display_name: "p1", __last_update: "2024-01-24 16:46:16" }];
```

Comment:
the `mapWowlValueToLegacy` affects `date` / `datetime` / `many2one` / `reference` / `one2many` / `many2many` fields. 

The x2many fields don't matter in our use case I think:

https://github.com/odoo/odoo/blob/7452f855f05251a5458ddb4d654b03be1b179d40/addons/web/static/src/views/debug_items.js#L232-L237

but I think it would be useful to ensure at least the value of other field type. There is a field "m2o" many2one on "partner" model. But we could add other fields with:

```js
serverData.models.partner.fields.date = {string: 'Date', type: 'date'}
serverData.models.partner.fields.reference = {string: 'Reference', type: 'reference'}
```
(and while we are at it, using `__last_update` is a little hacky since it's a field that's already a little hacky, so we could also add a normal datetime field)

Reply:
when the value is date its type gets passed as datetime too as its value gets changed to meet the ISO format too. So, I think we can leave it without a test case as the datetime test case covers both.